scripts/get_nearby_restaurants.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import json
+
+currentLocation = input('Search nearest restaurants in: ')
+
+urlBase = "https://maps.googleapis.com/maps/api/place/textsearch/json?query=restaurants+in+"+currentLocation+"&key="
+api_key = "" #Create your own API key or ask me for it.
```

Comment:
General comment, you don't need to take action on this. Good that you didn't push your API key. 

The canonical way to hide API keys is to have a hidden `.env` file somewhere that stores all your API keys and other credentials. You'd have your script read from this `.env` file and store your credentials in some constant so nobody can know your API key instead of leaving this as an empty string. Obviously, this `.env` file is never pushed to github or uploaded anywhere.